routes.js

```diff
@@ -15,6 +15,7 @@ const routes = require('next-routes')();
 // @RANStartRoutes
 
 routes.add('meeting', '/meeting/:meetingId');
+routes.add('terms', '/terms');
```

Comment:
This actually isn't needed! https://nextjs.org/learn/basics/navigate-between-pages